.github/workflows/code-checks.yml

```diff
@@ -13,6 +13,12 @@ concurrency:
 jobs:
   mypy:
     runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - {name: "app", pkg: "lightning_app"}
+          - {name: "pytorch", pkg: "pytorch_lightning"}
```

Comment:
I'm not seeing the new job run in the CI (not sure why)

Also this will need to update `checkgroup` :)

Reply:
nop, my bad, syntax error: https://github.com/Lightning-AI/lightning/actions/runs/2905195501